tests/unit/test_messages.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase
```

Comment:
We should consider using the pytest framework and not unittest since this is a new repo, a cleaner less boiler plate implementation which lets you use parameterization. This can be put on another card mind, we just don't want to do too many tests in the old format before we switch. Would like the whole of RAS/RM to use it, but that horse has bolted and probably died of old age